File: JS/human_readable_time.js
//https://www.codewars.com/kata/52685f7382004e774f0001f7/train/javascript
function humanReadable(seconds) {
    let hours = Math.floor(seconds / 3600);
    let minutes = Math.floor(seconds / 60) - (hours * 60);
    let secs = seconds - (hours * 3600) - (minutes * 60);

    let timeArray = [hours, minutes, secs];
    let timeString = "";
    timeArray.forEach(function(value) {
        if (value < 10){
            timeString += "0";
        }
        timeString += String(value) + ":";
    });
    return timeString.slice(0,-1);
}

File: Python/human_readable_time.py
#https://www.codewars.com/kata/52685f7382004e774f0001f7/train/python
def make_readable(seconds):
    hours = seconds // 3600
    minutes = seconds // 60 - (hours * 60)
    secs = seconds - (3600 * hours) - (60 * minutes)
    time_string = ""
    for item in [hours, minutes, secs]:
        if item < 10:
            time_string += "0"
        time_string += str(item) + ":"
    return(time_string[:-1])
